apps/jobs/models.py

```diff
@@ -83,6 +83,7 @@ class Submission(TimeStampedModel):
     method_description = models.TextField(blank=True, default="")
     publication_url = models.CharField(max_length=1000, default="", blank=True)
     project_url = models.CharField(max_length=1000, default="", blank=True)
+    baseline_submission = models.BooleanField(default=False, null=False)
```

Comment:
Is there any specific reason for `null=False`?

Reply:
I have considered `baseline_submission` to either be true or false **only**.

Reply:
yeah, but by default, it takes `null=False` if you don't specify it. Can you please remove it.

Reply:
@vkartik97 Any updates on this?